src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -311,14 +358,14 @@ private void WorkerLoop()
 
                         // We have one or more active operations. Run any work that needs to be run.
                         CURLMcode performResult;
-                        while ((performResult = Interop.Http.MultiPerform(multiHandle)) == CURLMcode.CURLM_CALL_MULTI_PERFORM);
+                        while ((performResult = Interop.Http.MultiPerform(_multiHandle)) == CURLMcode.CURLM_CALL_MULTI_PERFORM);
```

Comment:
I'd just add one before.  Just because the old flow was to make it as a local and the new flow is to trust the field.

But I agree that the exceptions would be indicative of what state things got into; so it's certainly ignorable.